platform/viewer/public/config/netlify.js

```diff
@@ -53,12 +53,12 @@ window.config = {
     // firstImage
     // lastImage
     {
-      commandName: 'nextViewportDisplaySet',
```

Comment:
Next was previous, previous was next  🤓

Reply:
Yeah. Can you see any reason for that? hehe as it was like this everywhere it make me think about it. haha